static/app/components/sidebar/sidebarItem.tsx

```diff
@@ -311,9 +319,28 @@ function SidebarItem({
 }
 
 export function isItemActive(
-  item: Pick<SidebarItemProps, 'to' | 'label'>,
+  item: Pick<SidebarItemProps, 'to' | 'label' | 'search'>,
   exact?: boolean
 ): boolean {
+  // issue subnav matches against issue groups in search params
+  if (location.pathname.startsWith('/issues/') && item?.to?.includes('/issues/')) {
+    const search = new URLSearchParams(location.search);
+    const itemSearch = new URLSearchParams(item.search);
+    const itemQuery = itemSearch.get('query');
+    const query = search.get('query');
+    if (item?.label === 'All') {
+      return !query && !itemQuery;
+    }
+    if (itemQuery && query) {
+      let match = false;
+      for (const key of itemQuery?.split(' ')) {
+        match = query.includes(key);
+        if (!match) break;
+      }
+      return match;
+    }
+    return !itemQuery;
+  }
```

Comment:
This is new "active" matching logic for the Issues subnav which required special consideration of search params